### ISSUE TITLE
Error for typescript.tsx buffers

### ISSUE THREAD

**Issue by 3rd** [kkoomen/vim-doge#76]
**Describe the bug**

When trying to generate documentation for a function in a `typescript.tsx` buffer, the following error appears:

```
[DoGe] "jsdoc" is not a valid typescript.tsx doc standard, available doc standard are: [] 
```

Related to: https://github.com/kkoomen/vim-doge/issues/73

**Settings**

```vim
let g:doge_filetype_aliases = {
      \  'javascript': [
      \    'javascript.jsx',
      \    'javascriptreact',
      \    'javascript.tsx',
      \    'typescriptreact',
      \    'typescript',
      \    'typescript.tsx',
      \  ],
      \  'java': ['groovy'],
      \}
```

**To Reproduce**

1. Open a `file.tsx` (`typescript.tsx`)
2. Try to generate the documentation of a function
3. Notice the error

**Expected behavior**

No error

**Screenshots**

![ss](https://i.imgur.com/g9vj6yA.png)

**Out of `vim --version`**
NVIM v0.5.0-271-g52566dd78

---

I'll try to do my part and fix it these days, thank you for your work!

**Comment by kkoomen** [comment]
It's a problem on my side. I will fix it today somewhere and I will add `typescript.tsx` to the default list of supported aliases. This should be supported by default and should not be set by you.

**Comment by kkoomen** [comment]
This bug has been fixed and released in [v2.2.1](https://github.com/kkoomen/vim-doge/tree/v2.2.1).

Feel free to submit any new issues if you experience any unwanted behavior in the future. Thanks for your contribution.
